system-test/bigquery.js

```diff
@@ -98,6 +98,31 @@ describe('BigQuery', function() {
     });
   });
 
+  it('should allow limiting API calls', function(done) {
+    var maxApiCalls = 1;
+    var numRequestsMade = 0;
+
+    var bigquery = gcloud.bigquery();
+
+    bigquery.interceptors.push({
+      request: function(reqOpts) {
+        numRequestsMade++;
+        return reqOpts;
+      }
+    });
+
+    bigquery.getDatasets({ maxApiCalls: maxApiCalls }, function(err) {
+      assert.ifError(err);
+
+      // Even though the request interceptor is called, the request can still be
+      // prevented if the `maxApiCalls` limit was reached.
+      numRequestsMade -= 1;
```

Comment:
Assuming it is kind of a pain to prevent, at the very least could we document this in the interceptors section? It probably won't affect anyone, but I think on the off chance some one is using this for capturing some kind of metric/etc., it might be good to have it documented somewhere.